rfcs/text/0003-object-field.md

```diff
@@ -1,30 +1,33 @@
-# 0003: Object Fields
+# 0003: Enterprise Content Fields (Previously known as Object Fields)
```

Comment:
When we get agreement on the renaming, we can also rename the file (e.g. `rfcs/text/0003-enterprise-content.md`)

Reply:
👍